cohort_4/playbook/ACTIVITIES.md

```diff
@@ -1431,3 +1432,21 @@ Want to send a link to everyone? _Slack._
 
 Was there an event at Galvanize that had some extra food and offered it to our
 hackers? _Slack._
+
+## Additional Resources
+
+### CodeHS
+
+#### Pros:
+
+- is very good at introducing and meticulously explaining the basic
+concepts of programming I (Jonathan) have not found anything clearer than
+CodeHS at explaining these basics.
+
+#### Cons:
+
+- the student nevers have the opportunity to build your own projects in CodeHS.
+  You are told everything that you are supposed to build.
+- perhaps even worse, you have no experience even in building _any_ kind
+  projects because you are always following a specific directions that keep
+  giving you the next step.
```

Comment:
@ssun098, I agree, @ssun098 can you take out the entire CodeHS section in this pull request? Thanks!